Updates docs for smart contracts (#8)
aragon/builders-portal: docs/smart-contracts/utils/AppStorage.md

*bytes32 internal constant DAO_POSITION = keccak256("core.storage.dao");*

#### internal variable `INITIALIZED_POSITION`

> The position in storage in which the boolean which detects if it's initialized or not.

Reviewer: ```suggestion
The position in storage where the boolean detects if it's initialized or not.
```

Reviewer: What is "it" in this context?

|:----- | ---- | ----------- |
| _dao | address | The address of the `IDAO` contract to be stored. |

#### internal function `initialized`

> Sets the initialized boolean to true..

Reviewer: ```suggestion
Sets the initialized boolean to true.
```

```solidity
function initialized() internal 
```

#### internal function `isInitialized`

```solidity
function isInitialized() internal view returns (bool) 
```

| Output | Type | Description |
| ------ | ---- | ----------- |

> | [0] | bool | bool whether it's initialized or not. |

Reviewer: ```suggestion
| [0] | bool | Checks whether the contract is initialized or not |
```
